data/tutorials/language/5rt_01_garbage-collector.md

```diff
@@ -0,0 +1,677 @@
+---
+id: garbage-collector
+title: Understanding the Garbage Collector
+description: >
+  Understanding the Garbage Collector, excerpt from Real World OCaml
+category: "Runtime & Compiler"
+external_tutorial:
+  tag: "RWO"
+  banner:
+    image: "tutorials/rwo_banner.png"
+    url: https://dev.realworldocaml.org/
+    alt: "Real World OCaml"
+  contribute_link:
+    url: https://github.com/realworldocaml/book/blob/master/book/garbage-collector/README.md
+    description: "You are encouraged to contribute to the original sources of this page at the Real World OCaml GitHub repository."
+---
+
+This is an adaptation of the chapter [Understanding the Garbage Collector](https://dev.realworldocaml.org/garbage-collector.html) from the book [Real World OCaml](https://dev.realworldocaml.org/), reproduced here with permission.
+
+
+# Understanding the Garbage Collector
+
+*This chapter includes contributions from Stephen Weeks and Sadiq Jaffer.*
+
+We've described the runtime format of individual OCaml variables earlier, in
+[Memory Representation Of Values](runtime-memory-layout.html#memory-representation-of-values){data-type=xref}.
```

Comment:
These cross references are RWO-specific -- probably want to edit them here.

Reply:
This one's an oversight. Fixed. (Also found another.) :+1: